### ISSUE TITLE
Can I Host this webpage

### ISSUE THREAD

**Issue by kartikvirendra** [jay1310/git_commands_collection#20]

**Comment by akshitsangwan** [comment]
@kartikvirendra Yes you can, just go to any free hosting provider such as Github Pages, Netlify, Heroku etc. and then add the link of the github repo you want to host.

**Comment by kartikvirendra** [comment]
yes I have hosted it using github pages - https://kartikvirendra.github.io/git_commands_collection/, please check pr #28
